### PR TITLE
Add anonymize plugin

### PR DESCRIPTION
**Pull request by korycins**
Add anonymize plugin which hides all user details.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for 82a123d83e061e342b164ec31562b3ddf7d9827b (korycins/saleor @ feature/anonymize_plugin)
Base comparison is 73e887eb8a9c47b77636880464c61ef4963a0600.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         14	         14	              0
  complete checkout                          	          6	          6	              0
  create checkout                            	         48	         48	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	         13	         13	              2

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: saleor/extensions/plugins/anonymize/plugin.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    def customer_created(self, customer: "User", previous_value: Any) -> Any:
+        customer.email = obfuscate_email(customer.email)
+        customer.save()
+
+    @classmethod
+    def _get_default_configuration(cls):
+        defaults = {
+            "name": cls.PLUGIN_NAME,
+            "description": (
+                "Hide all user data like shipping and billing addresses, email and "
```
Comment (maarcingebala) [inline saleor/extensions/plugins/anonymize/plugin.py]: "Anonymize customer's personal data in the checkout, such as shipping or billing address, email and phone number."

Comment (korycins) [reply saleor/extensions/plugins/anonymize/plugin.py]: 👮

File: saleor/core/utils/__init__.py [hunk 2]
```diff
@@ -212,3 +219,10 @@ def get_country_name_by_code(country_code):
 
 def get_company_address():
     return Site.objects.get_current().settings.company_address
+
+
+def get_random_avatar():
```
[the remark this answers is not in the document]
Comment (korycins) [reply saleor/core/utils/__init__.py]: 👮 @maarcingebala 👮 
Circular import. `Core` module shouldn't import dependencies from different modules. `get_random_avatar` is used in `core`  and `account/mutations`. 

EDIT
I am checking it now and I think that we could move `create_superuser` (it uses get_random_avatar) from `core.utils` module to `populatedb` logic - `create_superuser` is used only there.

[the remark this answers is not in the document]
Comment (maarcingebala) [reply saleor/core/utils/__init__.py]: Yep, I guess something else is causing the issue - `account/utils` is a perfect place for this function.

File: saleor/account/utils.py [hunk 3]
```diff
@@ -73,6 +77,25 @@ def get_user_last_name(user):
     return None
 
 
+def create_superuser(credentials):
```
Comment (NyanKiyoshi) [inline saleor/account/utils.py]: 👀 
```suggestion
def create_superuser(email, password):
```

**Pull request merged**